src/linux_ip_socket/linux_ip_socket.h

```diff
@@ -94,21 +95,14 @@ class linux_ip_socket_private_data final
     void driver_send(const uint8_t* data, const size_t length);
 
   private:
-    enum PacketParseState
-    {
-        STATE_WAIT,
-        STATE_DATA_BYTE,
-        STATE_ESCAPE_BYTE,
-    };
 
     static constexpr int DRIVER_THREAD_PRIORITY = 1;
     static constexpr int DRIVER_THREAD_STACK_SIZE = 65536;
     static constexpr int DRIVER_MAX_CONNECTIONS = 1;
-    static constexpr size_t DRIVER_SEND_BUFFER_SIZE = 8 * 1024;
-    static constexpr size_t DRIVER_RECV_BUFFER_SIZE = 8 * 1024;
-    static constexpr uint8_t START_BYTE = 0x00;
-    static constexpr uint8_t STOP_BYTE = 0xFF;
-    static constexpr uint8_t ESCAPE_BYTE = 0xFE;
+    static constexpr size_t DRIVER_RECV_BUFFER_SIZE = 1 * 1024;
```

Comment:
How are these sizes calculated? Do they take into account that encoded data could be twice big as the source data?

Reply:
It is made similar to SAMv71 Drivers project.

`#define Serial_CCSDS_SAMV71_RECV_BUFFER_SIZE 256
#define Serial_CCSDS_SAMV71_ENCODED_PACKET_MAX_SIZE 256
#define Serial_CCSDS_SAMV71_DECODED_PACKET_MAX_SIZE BROKER_BUFFER_SIZE`

Recv buffer has the same size as encoded buffer. Decoded buffer size is inherited from Broker.